submissions/xT0PERx/html-movie-seat-booking/main.css

```diff
@@ -115,98 +110,97 @@ html {
   text-transform: uppercase;
 }
 
-.calendarWrap {
+.calendar_wrap {
   display: grid;
   grid-template-columns: repeat(7, 1fr);
```

Comment:
You use 7 columns for 3 elements, if you need these proportions just use `1fr 5fr 1fr`.
Also, remove this code at line 125 `grid-column: span 5;` for same result.

![image](https://user-images.githubusercontent.com/16196199/107091290-38fd0600-680a-11eb-9fa5-2c495868908e.png)